guides/users/users/overview.md

```diff
@@ -0,0 +1,78 @@
+# Overview
+
+Solidus includes a straightforward user management system. Users are your stores
+customers, as well as your store administrators who need access to the Solidus
+admin interface.
+
+You can search and sort users using a few types of information:
+
+- **Email**: Users access your store and make orders using an identifying email
```

Comment:
Users sign into your store is more accurate as they can access the store without leaving an email